CONTRIBUTION.md

```diff
@@ -19,8 +19,8 @@ If any part of the project has bugs or documentation mistakes, please let us kno
 
 This is an outline of what a contributor's workflow looks like:
 
-- Fork the repository on GitHub
-- Clone the forked repository in your go path.
+- Fork the repository on GitHub.
+- Clone the forked repository in your go path, refer [kubernetes example](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#2-clone-fork-to-local-storage).
```

Comment:
Two things:
1) It should be `refer to` not `refer`
2) Is this even true? I don't have my fork in my go path, I don't believe you have to/ are recommended to, if you are using go mod.

Reply:
right, I  agree with that, the go mod should replace the business of dealing with the gopath. But the main intention of this is to not get confused with placing the fork in github.com/vinaykns/wmco vs github.com/openshift/wmco . To point that the preferred way is the second approach. 

How does it sound to replace with clone the fork to local storage, refer to (kubernetes example)..?

Reply:
The kubernetes example still says to copy it to your gopath. I'd just say clone the fork to local storage and leave out the kubernetes link.